### PR TITLE
mapping: use xmap if possible

### PR DESCRIPTION
**Pull request by tsuyoshicho**
Currently K is used vmap.

But, this command map to select-mode too.

This PR is using xmap if possible.

### CODE REVIEW

File: plugin/ref.vim [hunk 1]
```diff
@@ -20,7 +20,11 @@ vnoremap <silent> <Plug>(ref-keyword) :<C-u>call ref#K('visual')<CR>
 
 if !exists('g:ref_no_default_key_mappings') || !g:ref_no_default_key_mappings
   silent! nmap <silent> <unique> K <Plug>(ref-keyword)
-  silent! vmap <silent> <unique> K <Plug>(ref-keyword)
+  if exists(':xmap') == 2
```
Comment (thinca) [inline plugin/ref.vim:23]: I think this check is unnecessary.  Just use `:xmap` instead of `:vmap`.

**Review by thinca — CHANGES_REQUESTED** [review]

File: plugin/ref.vim (hunk 1 above)
Comment (tsuyoshicho) [reply plugin/ref.vim:23]: Okay, refine it.

Comment (tsuyoshicho) [reply plugin/ref.vim:23]: If can fix together `vnoremap` Plug in mapping (to `xnoremap`).

Do you require or not?

File: plugin/ref.vim [hunk 2]
```diff
@@ -20,7 +20,11 @@ vnoremap <silent> <Plug>(ref-keyword) :<C-u>call ref#K('visual')<CR>
```
Comment (tsuyoshicho) [inline plugin/ref.vim:20]:
```suggestion
xnoremap <silent> <Plug>(ref-keyword) :<C-u>call ref#K('visual')<CR
```

File: plugin/ref.vim (hunk 1 above)
Comment (tsuyoshicho) [reply plugin/ref.vim]: Fix together.

**Review by thinca — APPROVED** [review]
LGTM

**Comment by thinca** [comment]
Thank you :)

**Comment by tsuyoshicho** [comment]
Thanks you too.

**Pull request merged**